nucypher/blockchain/eth/actors.py

```diff
@@ -410,176 +432,102 @@ def total_staked(self) -> NU:
         else:
```

Comment:
Sorry, missed this before - this function sometimes will return incorrect values, because staked values is a function of period
Right value is `staked_amount` in `calculate_staking_reward`

Reply:
:heavy_check_mark: